pkg/templates/images/images.go

```diff
@@ -301,40 +301,47 @@ func optionalResources() map[Resource]map[string]string {
 
 		// OpenStack CCM
 		OpenstackCCM: {
-			"1.24.x":    "docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.24.5",
-			"1.25.x":    "docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.25.3",
-			">= 1.26.0": "docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.26.0",
+			"1.25.x": "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.25.6",
+			"1.26.x": "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.26.3",
+			"1.27.x": "registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.27.1",
```

Comment:
Please leave `>= 1.27.0` for (forward) compatibility reasons. There might be cases where CCM/CSI doesn't yet have a matching minor release, but it can be used as per the version skew policy. That's often the case for vSphere. We might also start using KubeOne to test prereleases in the future.

Reply:
AFAIK, openstack's CCM/CSI have hard binding against kubernetes version, and they don't want you to deploy CCM 1.27.x to kubernetes 1.28.x :shrug:

Reply:
As per the [version skew policy](https://kubernetes.io/releases/version-skew-policy/#kube-controller-manager-kube-scheduler-and-cloud-controller-manager), it's allowed. This is going to be very important once we start testing prereleases and I'm planning on adding that soon.

Reply:
done